orbit/pkg/platform/platform_notwindows.go

```diff
@@ -27,3 +28,33 @@ func ChmodExecutable(path string) error {
 	}
 	return nil
 }
+
+// GracefulProcessKillByName looks for all the process running under a given name
+// and force terminate them sending the SIGKILL signal
+func GracefulProcessKillByName(name string) error {
+	if name == "" {
+		return errors.New("process name should not be empty")
+	}
+
+	// Getting the target process to gracefully shutdown
+	foundProcess, _ := GetProcessByName(name)
+	if foundProcess == nil {
+		return nil // not an error, just no processes found
+	}
+
+	// Checking if target process is running
+	// and force kill it if this happens to be the case
+	isRunning, err := foundProcess.IsRunning()
+	if err != nil {
+		return fmt.Errorf("couldn't get running information on process %d: %w", foundProcess.Pid, err)
+	}
+
+	// Process is still running - force killing it
+	if isRunning {
+		if err := foundProcess.Kill(); err != nil {
```

Comment:
Please remind me, is this force killing it? (SIGKILL). If so, maybe method name should not include `Graceful`? :)

Reply:
yeah, good point. The method first attempt to signal the remote windows object by sending `WM_CLOSE` and then force kill the process if that fails. Idea is to have the entire logic centralized within a single method. I agree that the name is a bit misleading and that should change